dipy/workflows/segment.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import division, print_function, absolute_import
+
+from glob import glob
+from os.path import join, basename, splitext, dirname, isabs, exists
+from os import makedirs
+
+import nibabel as nib
+import numpy as np
+
+from dipy.segment.mask import median_otsu
+
+
+def median_otsu_bet(input_file, out_dir, save_masked=False, median_radius=4,
+                    numpass=4, autocrop=False, vol_idx=None, dilate=None):
+
+    for fpath in glob(input_file):
+        img = nib.load(fpath)
+        volume = img.get_data()
+        masked, mask = median_otsu(volume, median_radius, numpass, autocrop,
+                                   vol_idx, dilate)
+
+        fname, ext = splitext(basename(fpath))
+        if(fname.endswith('.nii')):
+            fname, _ = splitext(fname)
+            ext = '.nii.gz'
+
+        mask_fname = fname + '_mask' + ext
+
+        if out_dir == '':
```

Comment:
Okay so here you say that if empty string then use the directory of fpath. Nice.